README.md

```diff
@@ -57,4 +29,4 @@ FAST-DNA has partnered with Deque & Microsoft to build accessibility into our co
 ## Contact
 * Submit [for help](https://stackoverflow.com/questions/tagged/fast-dna) 
 * Submit [feature requests](https://github.com/Microsoft/fast-dna/issues/new?labels=feature%20:%20request)
-* Submit [feature questions](https://github.com/Microsoft/fast-dna/issues/new?labels=feature%20:%20question)
\ No newline at end of file
+* Submit [feature questions](https://github.com/Microsoft/fast-dna/issues/new?labels=feature%20:%20question)
```

Comment:
what is a feature question?

Reply:
when someone from the community has a question about how a feature works, usually, when they are contemplating contributing in some manner.

Reply:
Can we just go with "Questions" and link there? Feature question would cause me to think that is where I send feature requests

Reply:
That would be "feature : request" and already exists

Reply:
We could do something where it maybe it says "help : question" "help : request" "help : wanted" ?

Reply:
This then maintains similar tagging structure and provides broader support?  could also be "triage : question" or "triage : request" ?  I might actually like this more.

Reply:
I like "help : question"